lpr.Logic/Services/OrganisationService.cs

```diff
@@ -18,22 +18,37 @@ public class OrganisationService : IOrganisationService {
       _organisationData = new OrganisationData(ctx);
     }
 
-    public int AddOrganisation(string Name, string UserId) {
+    public void AddOrganisation(string Name, string UserId) {
```

Comment:
Maybe return something here, to confirm that AddOrganisation actually got processed.